runtime/fiber.c

```diff
@@ -18,8 +18,11 @@
 
 #define CAML_INTERNALS
 
+#include "caml/config.h"
```

Comment:
Can you say a few words about why this included was added?

Reply:
It's where `HAS_UNISTD` is defined. If no `caml/*.h` header is included before C or system headers are included, then we don't get this macro. From the Autoconf manual

> The package should ‘`#include`’ the configuration header file before any other header files, to prevent inconsistencies in declarations (for example, if it redefines `const`, or if it defines a macro like `_FILE_OFFSET_BITS` that affects the behavior of system headers). 

https://www.gnu.org/software/autoconf/manual/autoconf-2.71/html_node/Configuration-Headers.html

Reply:
OK, thanks.

Reply:
I've added a short note about this in the commit message.